tests/responses/refund_single.json

```diff
@@ -14,37 +14,37 @@
   "description": "Order",
   "paymentId": "tr_7UhSN1zuXS",
   "lines": [
```

Comment:
Please take another look at the documentation; the examples have been updated for the orders and shipments API's.